moped-database/migrations/1700149643858_parent_audit_values/up.sql

```diff
@@ -0,0 +1,69 @@
+-- Altering moped_project table to add a new column for tracking the user who last updated a record
+ALTER TABLE moped_project ADD COLUMN updated_by_user_id INTEGER DEFAULT NULL;
+COMMENT ON COLUMN moped_project.updated_by_user_id IS 'User ID of the person who last updated the project';
```

Comment:
i just want to note that we have https://github.com/cityofaustin/atd-data-tech/issues/14564 to cover the task of setting up a Hasura column preset that should populate the `moped_project.updated_by_user_id` column for any project edit.